base/shell/explorer/lang/pt-PT.rc

```diff
@@ -0,0 +1,203 @@
+/* Translation and UTF-8 Conversion by mkbu95 <mkbu95@gmail.com> (May, 2012) */
+
+LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL
```

Comment:
This now duplicates what's in pt-BR.rc -- if you want this file to cover SUBLANG_NEUTRAL, you'll need to change the sub-language for pt-BR.
You can see these errors in CI that are indicative of the problem:
```
c:\reactos-cov\base\shell\explorer\lang/pt-PT.rc(192) : error RC2151 : cannot reuse string constants, 595(0x253) - "Iniciar" already defined. Cannot add 595(0x253) - "Iniciar".
c:\reactos-cov\base\shell\explorer\lang/pt-PT.rc(193) : error RC2151 : cannot reuse string constants, 720(0x2D0) - "P&ropriedades" already defined. Cannot add 720(0x2D0) - "P&ropriedades".
c:\reactos-cov\base\shell\explorer\lang/pt-PT.rc(194) : error RC2151 : cannot reuse string constants, 718(0x2CE) - "A&brir Todos os Usußrios" already defined. Cannot add 718(0x2CE) - "A&brir Todos os Usußrios".
c:\reactos-cov\base\shell\explorer\lang/pt-PT.rc(195) : error RC2151 : cannot reuse string constants, 719(0x2CF) - "E&xplorar Todos os Usußrios" already defined. Cannot add 719(0x2CF) - "E&xplorar Todos os Usußrios".
c:\reactos-cov\base\shell\explorer\lang/pt-PT.rc(196) : error RC2151 : cannot reuse string constants, 105(0x69) - "O sistema nπo pode iniciar o explorer porque o registro estß corrompido ou inacessφvel." already defined. Cannot add 105(0x69) - "O sistema nπo pode iniciar o explorer porque o registo estß corrompido ou inacessφvel.".
c:\reactos-cov\base\shell\explorer\lang/pt-PT.rc(201) : error RC2151 : cannot reuse string constants, 810(0x32A) - "Propriedades do Barra de Tarefas e Menu Iniciar" already defined. Cannot add 810(0x32A) - "Propriedades do Barra de Tarefas e Menu Iniciar".
c:\reactos-cov\base\shell\explorer\lang/pt-PT.rc(202) : error RC2151 : cannot reuse string constants, 811(0x32B) - "&Show Open Windows" already defined. Cannot add 811(0x32B) - "&Mostrar janelas abertas".
```

Reply:
In principle, the Portuguese brazilian (pt-BR) should be:
`LANGUAGE LANG_PORTUGUESE, SUBLANG_PORTUGUESE_BRAZILIAN`
while the Portuguese from Portugal (pt-PT) is:
`LANGUAGE LANG_PORTUGUESE, SUBLANG_NEUTRAL`